multilabel_classification/pytorch_network_training.ipynb

```diff
@@ -0,0 +1,1064 @@
+    "        dataset_val,\n",
+    "        input_size=data_config[\"input_size\"],\n",
+    "        batch_size=64,\n",
+    "        is_training=False,\n",
+    "        mean=data_config[\"mean\"],\n",
+    "        std=data_config[\"std\"],\n",
+    "        interpolation=data_config[\"interpolation\"],\n",
+    "\n",
+    "    )\n",
+    "    model.fit(loader_train,loader_val,num_epochs=40)\n",
+    "    checkpoint = torch.load(\"weights_model/model_best.pth.tar\")\n",
+    "    model.load_state_dict(checkpoint['state_dict'])\n",
+    "    pred_val = model.predict_proba(loader_val)\n",
+    "    df_pred = create_df(pred_val,dataset_val)\n",
+    "    df_pred.to_csv(str(ct)+\"_fold.csv\",index=False)\n",
+    "    ct+=1\n",
+    "    \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<module 'os' from '/usr/lib/python3.8/os.py'>"
+      ]
+     },
+     "execution_count": 20,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "os"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dfl = []\n",
+    "for i in range(1,num_splits+1):\n",
+    "    dfl.append(pd.read_csv(str(i)+\"_fold.csv\"))\n",
+    "    \n",
+    "\n",
+    "cols = dfl[0].columns[1:]\n",
+    "\n",
+    "df_pred = pd.concat(dfl,axis=0)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pred_probs = df_pred[cols].to_numpy()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "np.save(open('pred_probs.npy','wb'),pred_probs)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 36,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = list(df_pred['image_loc'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 38,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<module 'pickle' from '/usr/lib/python3.8/pickle.py'>"
+      ]
+     },
+     "execution_count": 38,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "pickle.dump(data,open(\"data.p\",'wb'))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.5"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 4
+}
```

Comment:
Don't worry about polishing the text too much, I'll do a rewrite of the text for you to make the notebook as clear as possible